### ISSUE TITLE
Getting started: NVDA does not allow up / down navigation

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#117448]
Refs: #117313

1. Turn on NVDA screen reader on windows
2. VS Code - > getting started
3. Try to use up / down to navigate between getting started buttons -> does not work 🐛 

NVDA seems to be eating up the up and down keys.
I am not sure why that is the case. Probably because NVDA thinks you are in Browse mode and then does not allow up / down.
@MarcoZehe might have an idea

**Comment by MarcoZehe** [comment]
I haven't tried it on Windows, but on Mac, this is definitely announced as a document. And this makes sense, since only traversing it in virtual mode guarantees we get to all the information that is there. I use VoiceOver navigation commands on that page/in that tab to get to all these bits of information as well. I am actually not sure this is a bug.

**Comment by isidorn** [comment]
Yeah on Mac it worked just fine for me. @JacksonKearl can you double check if you also see this issue with Win and NVDA?

**Comment by JacksonKearl** [comment]
Testing on NVDA it does indeed swallow my up/down arrows, but I'm not sure if this is incorrect -- Up/Down move the virtual cursor through lines in the document and correctly announce things like the clickable headers, once a header is clicked up/down can navigate through the contents of that item. 

I believe this is NVDA's "browse mode", which should be able to be toggled off, unfortunately it seems my keyboard/VM do not allow me to enter the correct key combination to test this.

**Comment by isidorn** [comment]
@JacksonKearl yes I believe you are correct. Though there might be way to make NVDA switch automaticaly out of browse mode. Though for the getting started browse mode might be fine, so let's stick with this behavior unless users complain
